.github/workflows/stale.yaml

```diff
@@ -0,0 +1,21 @@
+name: "Close stale issues"
+on:
+  schedule:
+    - cron: "0 0 * * *"
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v4
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          stale-issue-message: 'Thank you for your contribution! This issue is stale because it has been open 60 days with no activity. In order to keep it open, please remove stale label or add a comment within the next 10 days.'
+          close-issue-message: 'Issue got stale and no further activity happend. It has automatically been closed. Please re-open in case you still consider it relevant.'
+          stale-pr-message: 'Thank you for your contribution! This pull request is stale because it has been open 60 days with no activity. In order to keep it open, please remove stale label or add a comment within the next 10 days.'
```

Comment:
```suggestion
          stale-pr-message: 'Thank you for your contribution! This pull request is stale because it has been open 60 days with no activity. In order to keep it open, please remove stale label or add a comment within the next 10 days. If you need a Piper team member to remove the stale label make sure to add '@SAP/jenkins-library-team' to your comment.'
```